RecoCTPPS/PixelLocal/BuildFile.xml

```diff
@@ -0,0 +1,20 @@
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/Framework"/>
+
+<use name="DataFormats/Common"/>
+<use name="DataFormats/CTPPSDigi"/>
+<use name="DataFormats/CTPPSReco"/>
+
+<use name="Geometry/VeryForwardGeometryBuilder"/>
+<use name="CondFormats/CTPPSReadoutObjects"/>
+
+<use name="DataFormats/CTPPSDetId"/>
+<use name="CondTools/CTPPS"/>
+<use name="CondCore/DBOutputService"/>
+<use name="FWCore/ServiceRegistry"/>
+
+<use name="root"/>
```

Comment:
you should not need the entire root external dependency - rather one of the partial components (eg, rootcore) What functionality are you using (I didn't find it)